scripts/build-installer-nix.sh

```diff
@@ -14,10 +14,20 @@ echo '~~~ Building mainnet installer'
 nix-build release.nix -A mainnet.installer --argstr buildNr $BUILDKITE_BUILD_NUMBER --argstr version $VERSION
 if [ -n "${BUILDKITE_JOB_ID:-}" ]; then
   buildkite-agent artifact upload result/Daedalus*installer*.bin --job $BUILDKITE_JOB_ID
+  daedalus_config=$(nix-build -A daedalus-config --no-out-link                           ./default.nix)
+  for cf in launcher-config wallet-topology
+  do cp ${daedalus_config}/$cf.yaml  $cf-mainnet.linux.yaml
+     buildkite-agent artifact upload $cf-mainnet.linux.yaml --job $BUILDKITE_JOB_ID
+  done
 fi
 
 echo '~~~ Building staging installer'
 nix-build release.nix -A staging.installer --argstr buildNr $BUILDKITE_BUILD_NUMBER --argstr version $VERSION
 if [ -n "${BUILDKITE_JOB_ID:-}" ]; then
   buildkite-agent artifact upload result/Daedalus*installer*.bin --job $BUILDKITE_JOB_ID
+  daedalus_config=$(nix-build -A daedalus-config --no-out-link --argstr cluster staging ./default.nix)
+  for cf in launcher-config wallet-topology
+  do cp ${daedalus_config}/$cf.yaml  $cf-staging.linux.yaml
+     buildkite-agent artifact upload $cf-staging.linux.yaml --job $BUILDKITE_JOB_ID
+  done
```

Comment:
it may be cleaner to change this whole thing into a bash function, and then just call it twice, so the code isnt duplicated

Reply:
I wondered why this factoring didn't happen originally, almost did that change : -)